### ISSUE TITLE
interface 'wl_output' has no event 4 when trying to launch Joplin on wayland native

### ISSUE THREAD

**Issue by AlvaroParker** [laurent22/joplin#6940]
## Environment

Joplin version: 2.8.8
Platform: Linux
OS specifics: Fedora Linux 36 (Using Flatpak)

## Steps to reproduce

1. Open terminal and run ` flatpak run net.cozic.joplin_desktop --enable-features=UseOzonePlatform --ozone-platform=wayland`

## Describe what you expected to happen
Launch normally with wayland support just like a Chromium web browser would under the same flags. 

## Logfile
 ```
LaunchProcess: failed to execvp:
xdg-settings
interface 'wl_output' has no event 4
```

**Comment by CalebJohn** [comment]
@AlvaroParker have you tried with the [latest AppImage](https://github.com/laurent22/joplin/releases/tag/v2.8.8)? The flatpack is created and published by a third party so we don't have the ability to fix flatpack specific issues here. You'll most likely want to report this issue to the flatpack maintainer.

Of course, if you have the same trouble with the AppImage we can help you out here.

**Comment by AlvaroParker** [comment]
> @AlvaroParker have you tried with the [latest AppImage](https://github.com/laurent22/joplin/releases/tag/v2.8.8)? The flatpack is created and published by a third party so we don't have the ability to fix flatpack specific issues here. You'll most likely want to report this issue to the flatpack maintainer.
> 
> Of course, if you have the same trouble with the AppImage we can help you out here.

Same behavior with the AppImage. When I run `./Joplin-2.8.8.AppImage --enable-features=UseOzonePlatform --ozone-platform=wayland` the output is
```
xdg-settings: invalid application name
Try 'xdg-settings --help' for more information.
interface 'wl_output' has no event 4
```
The first 2 lines of the output appears even without the arguments `--enable-features=UseOzonePlatform --ozone-platform=wayland`
Edit: With AppImage v2.9.4. the problem is fixed.

**Comment by CalebJohn** [comment]
Thanks for checking with the AppImage and confirming that this is fixed in the upcoming release. I didn't see your edit right away and looked into it. This is caused by an [electron bug](https://github.com/electron/electron/issues/32487) (which in turn is a chromium bug), it just takes awhile for the bugfix to go chromium -> Electron -> Joplin -> User. That said, it was fixed in https://github.com/laurent22/joplin/commit/27ef9173507318538c967d0d2c1aec6b794a8c5d and will be available in Joplin v2.9.x and above.
